src/lib/util.ts

```diff
@@ -64,3 +79,26 @@ export async function canvasEncode(data: ImageData, type: string, quality?: numb
   if (!blob) throw Error('Encoding failed');
   return blob;
 }
+
+export function canDecode(data: string): Promise<boolean> {
+  return new Promise((resolve) => {
+    const img = document.createElement('img');
+    img.src = data;
+    img.onload = _ => resolve(true);
+    img.onerror = _ => resolve(false);
+  });
+}
+
+export function fileToBitmap(file: File): Promise<ImageBitmap> {
+  return createImageBitmap(file);
+}
```

Comment:
This seems weird, but I think it's worth keeping as I don't think `createImageBitmap` is supported on our target browsers. So, we'll eventually expand this method to fall back to img + canvas or whatever.

Reply:
That was the point behind this function